code/modules/mob/living/simple_animal/hostile/terror_spiders/actions.dm

```diff
@@ -0,0 +1,127 @@
+
+
+/datum/action/innate/terrorspider/web
+	name = "Web"
+	icon_icon = 'icons/effects/effects.dmi'
+	button_icon_state = "stickyweb1"
+
+/datum/action/innate/terrorspider/web/Activate()
+	var/mob/living/simple_animal/hostile/poison/terror_spider/user = owner
+	user.Web()
+
+/datum/action/innate/terrorspider/wrap
+	name = "Wrap"
+	icon_icon = 'icons/effects/effects.dmi'
+	button_icon_state = "cocoon_large1"
+
+/datum/action/innate/terrorspider/wrap/Activate()
+	var/mob/living/simple_animal/hostile/poison/terror_spider/user = owner
+	user.FindWrapTarget()
+	user.DoWrap()
+
+
+// ---------- WEB
+
+/mob/living/simple_animal/hostile/poison/terror_spider/proc/Web()
+	visible_message("<span class='notice'>\the [src] begins to secrete a sticky substance.</span>")
```

Comment:
`\the` is unnecessary here, `[src]` will automatically prepend `The` if necessary.